Bottom: ca5242afb49b7dd2af565becf109403c062b28a9
Top:    2a23bb504e7919134c657861ae402ed215f628c2
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 14:18:37 +0800

Refresh of debug.patch

---

diff --git a/lib/rpc_subs.c b/lib/rpc_subs.c
index cfdbe29..6ac8d32 100644
--- a/lib/rpc_subs.c
+++ b/lib/rpc_subs.c
@@ -590,7 +590,7 @@ int rpc_portmap_getclient(struct conn_info *info,
 	info->addr = addr;
 	info->addr_len = addr_len;
 	info->program = rpc_getrpcbyname(rpcb_prog);
-	info->port = rpc_getrpcbport(proto)
+	info->port = rpc_getrpcbport(proto);
 	info->version = rpcb_version;
 	info->proto = proto;
 	info->send_sz = RPCSMALLMSGSIZE;
